### ISSUE TITLE
Routine update of dependencies

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2849]
We have secondary dependencies (dependencies of our pyproject.toml defined first order dependencies - variously pinned) that are in need of a refresh via our Poetry dependencies manager.

The following:
```
cd /opt/rockstor
poetry update
```
should accomplish this, producing an updated poetry.lock file; to be presented as the resolution of this issue.

---
Upstream reference: https://python-poetry.org/docs/basic-usage/#updating-dependencies-to-their-latest-versions

**Comment by phillxnet** [comment]
# Resulting Poetry report

```
Package operations: 1 install, 19 updates, 0 removals

  • Updating backports-tarfile (1.1.0 -> 1.2.0)
  • Updating cryptography (42.0.5 -> 42.0.8)
  • Updating more-itertools (10.2.0 -> 10.3.0)
  • Updating zipp (3.18.1 -> 3.19.2)
  • Updating certifi (2024.2.2 -> 2024.6.2)
  • Updating jaraco-functools (4.0.0 -> 4.0.1)
  • Updating setuptools (69.5.1 -> 70.0.0)
  • Updating typing-extensions (4.11.0 -> 4.12.2)
  • Updating urllib3 (2.2.1 -> 2.2.2)
  • Updating django (4.2.11 -> 4.2.13)
  • Updating keyring (25.1.0 -> 25.2.1)
  • Updating packaging (24.0 -> 24.1)
  • Installing pytz (2024.1)
  • Updating requests (2.31.0 -> 2.32.3)
  • Updating zope-interface (6.3 -> 6.4.post2)
  • Updating django-oauth-toolkit (2.3.0 -> 2.4.0)
  • Updating django-pipeline (3.0.0 -> 3.1.0)
  • Updating huey (2.5.0 -> 2.5.1)
  • Updating psycopg (3.1.18 -> 3.1.19)
  • Updating pyzmq (26.0.0 -> 26.0.3)

Writing lock file
```

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2851

**Comment by phillxnet** [comment]
N.B. It looks like the following update
 • Updating django-oauth-toolkit (2.3.0 -> 2.4.0)
brings the following upstream db migrations:

from rockstor.log:
```
[17/Jun/2024 19:06:48] INFO [scripts.initrock:646] Prior migrations for oauth2_provider are: ['oauth2_provider', ' [X] 0001_initial', ' [X] 0002_auto_20190406_1805', ' [X] 0003_auto_20201211_1314', ' [X] 0004_auto_20200902_2022', ' [X] 0005_auto_20211222_2352', ' [X] 0006_alter_application_client_secret', ' [X] 0007_application_post_logout_redirect_uris', ' [ ] 0008_alter_accesstoken_token', ' [ ] 0009_add_hash_client_secret', ' [ ] 0010_application_allowed_origins', '']
[17/Jun/2024 19:06:54] INFO [scripts.initrock:654] Post migrations for oauth2_provider are: ['oauth2_provider', ' [X] 0001_initial', ' [X] 0002_auto_20190406_1805', ' [X] 0003_auto_20201211_1314', ' [X] 0004_auto_20200902_2022', ' [X] 0005_auto_20211222_2352', ' [X] 0006_alter_application_client_secret', ' [X] 0007_application_post_logout_redirect_uris', ' [X] 0008_alter_accesstoken_token', ' [X] 0009_add_hash_client_secret', ' [X] 0010_application_allowed_origins', '']
```
These looks to have been applied successfully via initirock.py, but noting here in case we have folks encountering interrupted rockstor-build service runs. The above log was taken from a 5.0.9-0 to 5.0.10-0 Web-UI initiated update (from a signed local repo prior to PRODUCTION publishing of 5.0.9-10).
